linear_x: 0.0
linear_y: 0.2
angular_z: 0.0
duration: 1.0